backend/config/postgres-integration.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/go-pg/pg/v10"
+)
+
+// function to connect to Postgres
+func Connect2db() *pg.DB {
```

Comment:
I think this can change this to `connectToDatabase()`, as numbers are not normally used as verbs in methods names in Go.